nextflow_schema.json

```diff
@@ -117,8 +112,7 @@
                 "RunOnCpu": {
                     "type": "string",
                     "description": "Specifies whether to run 'colabfold' mode using cpus",
-                    "default": " ",
-                    "enum": [" ", "--cpu"]
```

Comment:
I think we can keep this line in order to let the users know the right formulation of the flag (at least until we find a more elegant way to provide this argument ;) )